Api/Api.csproj

```diff
@@ -8,6 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="Azure.Storage.Blobs" Version="12.18.0-beta.1" />
```

Comment:
This reference should be only in Infrastructure project and also do not use Beta package version